example/themes/original/src/components/OrderDetails/index.tsx

```diff
@@ -356,6 +365,25 @@ export const OrderDetailsUI = (props: OrderDetailsParams) => {
     )
   }
 
+  const handleReorder = async (orderId: number) => {
```

Comment:
Add this function in functional component from ordering-components, in order to reuse the function in another apps

Reply:
moved function to ordering component